client/src/Categories/CategoriesSources.js

```diff
@@ -0,0 +1,30 @@
+import { Link } from 'react-router-dom';
+import { useParams } from 'react-router-dom';
+
+import './CategoriesList.scss';
+import { useEffect, useState } from 'react';
+import Api from '../Api';
+
+function CategoriesList() {
+  const { slug } = useParams();
+  const [category, setCategory] = useState({});
+  // const [resources, setResources] = useState([]);
```

Comment:
I think it's still correct that we want to show some/all resources on this page, even if we don't name it as such in the url...